schemas/yazi.json

```
@@ -160,7 +156,12 @@
 				"cd_offset": {
 					"$ref": "#/$defs/offset"
 				},
-				"create_title": { "type": "string" },
+				"create_title": {
+					"type": "array",
+					"items": { "type": "string" },
+					"minItems": 2,
+					"maxItems": 2
+				},
```

create_title can still be a single string - if you look at the diff (https://github.com/sxyazi/yazi/commit/dfe2884110d24515b591192be94b17c44689015d#diff-86ad26ab85d77e4bd594a4813c46c5292a93e9ddfc9f4c224e3b9d8627635b74, `yazi-config/src/popup/input.rs`), you'll see:
```rs
pub enum InputCreateTitle {
	One(String),
	Two([String; 2]),
}
```

This should work - fix the spaces in the formatting though. Make sure to test it!
```suggestion
				"create_title": {
			        "oneOf": [
			          {
			            "type": "array",
			            "items": { "type": "string" },
			            "minItems": 2,
			            "maxItems": 2
			          },
			          {
			            "type": "string"
			          }
			        ]
				},
```